src/utils/resolveToValue.js

```diff
@@ -107,14 +138,18 @@ function findLastAssignedValue(scope, name) {
  *
  * Else the path itself is returned.
  */
-export default function resolveToValue(path: NodePath): NodePath {
+export default function resolveToValue(
+  path: NodePath,
+  resolveImports: boolean = true,
```

Comment:
```suggestion
  resolveImports?: boolean = true,
```

I wonder if it makes more sense to have `false` as default? That would be less breaking for outside resolvers/handlers. But we are also doing a major anyway...
Are we using more cases with `true` in our codebase?